interfaces/css-conditional.idl

```diff
@@ -27,7 +28,7 @@ interface CSSMediaRule : CSSConditionRule {
 interface CSSSupportsRule : CSSConditionRule {
 };
 
-partial interface CSS {
-  static boolean supports(CSSOMString property, CSSOMString value);
-  static boolean supports(CSSOMString conditionText);
+partial namespace CSS {
```

Comment:
cssom.idl still has interface CSS, so this will break, right?

Reply:
Not any more :)